src/converse-chatview.js

```diff
@@ -1018,6 +1026,25 @@ converse.plugins.add('converse-chatview', {
                 return this;
             },
 
+            async inject (ev) {
+                if (!_converse.debug) {
+                    return this;
+                }
+                const message_handler = _converse.chatboxes.onMessage.bind(_converse.chatboxes);
+                const msgs = new DOMParser().parseFromString('<s>'+ev+'</s>', 'text/xml').children[0].children;
```

Comment:
`firstElementChild` might be better than `children[0]`. I think `children[0]` might return a text node or other non-Element nodes.